docs/UserGuide.md

```diff
@@ -11,117 +11,287 @@
+Examples:
+* `rsv Meet John Doe -dt 26/09/2016 0900 to 1030, 28/09/2016 1000 to 1130`
 
-Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+#### Editing a reserved timeslot : `rsv -e`
+Renames a task with reserved time slots or Adds/Deletes a reserved timeslot for a task <br>
+Format: `rsv -e <INDEX> -n <TASK> -dta <START_DATE/TIME> to <END_DATE/TIME> -dtr <START_DATE/TIME> to <END_DATE/TIME>`
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+> Words in `UPPER_CASE` are the parameters. 
+> 
+> Time is in 24-hr format. More than one date/time can be edited.
+>
+> Use -n to rename the task <br>
+> Use -dta to add a timeslot <br>
+> Use -dtr to remove a timeslot <br>
 
-> Deletes the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+Examples:
+* `rsv -e 2 -n Meet John Tan -dta 08/10/2016 1000 to 1200`
+* `rsv -e 1 -n -dta 09/10/2016 1100 to 1230 -dtr 05/10/2016 1000 to 1200`
 
-Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+#### Deleting a task with reserved timeslots : `rsv -d`
+Deletes a task with all its reserved time slots <br>
+Format: `rsv -d <INDEX>`
+
+> Deletes the task at the specific `<INDEX>`. 
+> The index refers to the index number shown in the task list.
+> The index **must be a positive integer** 1, 2, 3, ...
+
+Examples:
+* `rsv -d 5`
+
+#### Confirming a reserved timeslot : `confirm`
+Confirms a reserved timeslot for a particular tasks and removed all the other reserved time slots. <br>
+Format: `confirm <INDEX_TASK> <INDEX_TIMESLOT>`
+
+> Confirm the task of a specific `<INDEX_TASK>` at a timeslot of a specific `<INDEX_TIMESLOT>` <br>
+> The `<INDEX_TASK>` refers to the index number shown in the task list. <br>
+> The `<INDEX_TIMESLOT>` refers to the index number of the timeslot shown under each task. <br>
+> Both indexes **must be a positive integer** 1, 2, 3, ...
+
+Examples:
+* `confirm 3 2`
+
+#### Editing a task : `edit`
+Edits any component of a particular task <br>
+Format: `edit <INDEX> -n <TASK> -dt <START_DATE/TIME> to <END_DATE/TIME> -p <PRIORITY> -t <TAG(s)`
+
+> Edits the task at the specific `<INDEX>`. 
+> The index refers to the index number shown in the task list.
+> The index **must be a positive integer** 1, 2, 3, ... 
+>
+> Words in `UPPER_CASE` are the parameters. Other than `<INDEX>`, all parameters are optional. <br>
+> Order of parameters are **not** fixed.
+>
+> Time is in 24-hr format (e.g. 1330)
+
+Examples:
+* `edit 3 -n Meet John Tan -dt 08/10/2016 1000 to 1200 -p H -t friend`
+
+#### Editing a task by appending details to a task : `edit -ap`
```

Comment:
This might be too complication for norm users, remember the learning curve that your project/app might have ~ perhaps this is only application to advance users?